tests/holdings_maintenance/test_task_flags.py

```
@@ -0,0 +1,104 @@
+        run: The ToolRun to read.
+
+    Returns:
+        str: The task the tool actually ran.
+    """
+
+    for line in run.output.splitlines():
+        _, sep, tail = line.partition('| HEADER | Task ')
+        if sep and ' for' in tail:
+            return tail.partition(' for')[0].strip('" ')
+
+    raise AssertionError(f'no task header in output\n{run.describe()}')
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)